platform/qt/resources/common.qrc

```diff
@@ -1,5 +1,6 @@
 <RCC>
     <qresource prefix="/">
+        <file alias="icon.png">../../../common/icon.png</file>
```

Comment:
Not really unrelated, this was on the same `qmapbox.qrc` file the ca-bundle used to be. I removed this `qmapbox.qrc` altogether and moved the icon to the `commom.qrc` shared by the examples, where the icon is used.